packages/web/src/custom.scss

```diff
@@ -31,7 +31,7 @@ $code-font-size: 100%;
     }
   }
 
-  .password-not-match{
+  .form-error-message{
     color: $color-lightcoral;
```

Comment:
How about make $color-lightcoral as an 'danger' color in out theme?

Reply:
fixed